static/css/main.css

```diff
@@ -1,5 +1,5 @@
 body {
-  padding-top: 20px;
+  /* padding-top: 20px; by alantai */ 
```

Comment:
Don't annotate your name here since we have version control systems like `Git`, that implies we can always track every change via commands like `git blame`.